### ISSUE TITLE
Submit Gamma 2016 to ArXiv

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-sky#56]
@vorugantia - this is a reminder issue to submit the Gamma 2016 to ArXiv, after deploying version 1.0 of gamma-sky.net

Select "LaTeX" version and upload this single file:
https://github.com/gammapy/gammaskynet-gamma2016/blob/master/proceeding/gammaskynet-gamma2016.tar.gz

We can use this issue to discuss if there are any questions or problems.

**Comment by cdeil** [comment]
@adonath and I just now found out that the gamma-sky.net proceeding is online and not behind a paywall:
http://aip.scitation.org/doi/pdf/10.1063/1.4969002

Apparently it's not included in ADS yet, but presumably this will automatically happen in the coming weeks:
https://ui.adsabs.harvard.edu/#search/fq=%7B!type%3Daqp%20v%3D%24fq_author%7D&fq_author=(author_facet_hier%3A%220%2FDeil%2C%20C%22)&q=deil&sort=date%20desc%2C%20bibcode%20desc

I don't think posting it on ArXiv now is useful. That is a preprint server that is mostly for things that aren't published yet or are behind a paywall. In this case it's published and freely available.

@adonath @vorugantia - Agreed or do you see a point in posting to ArXiv?

@vorugantia - Can you update the docs to mention this paper very prominently at the top?
I will try to review our docs and put "please cite us in this way if you've used gamma-sky.net" in the coming days, but for now just mentioning the paper is good enough, and then this issue can be closed.

**Comment by vorugantia** [comment]
I agree that we don't need to post to ArXiv anymore. I'll update the docs now.

**Comment by cdeil** [comment]
Thanks!
Closing this issue now.

**Comment by cdeil** [comment]
I'm re-opening this issue.

So what happened is that AIP gave free access for a month, but from now on (indefinitely?) the PDF is behind a paywall. Frank suggested we post on ArXiv, and we should do that.

@vorugantia - I would still suggest we work on the webpage a bit more. To be concrete, I think we should have this in place when we post on ArXiv (that's going to be the day when most astronomers look at it):

- URLs should be stable (keep or remove the hash symbol?)
- Gamma-cat should be updated to a version with more data (I just added a bunch of stuff yesterday, this is coming together)
- There shouldn't be so many "TODO" and a bit more info on the pages displaying info from the catalogs.

There's many other things (more catalogs, more images, links between source detail views an map view, ...) that can come later, but there should be a nice core page in place.

**Comment by cdeil** [comment]
It's here now: https://arxiv.org/abs/1709.04217

Since the journal proceeding is still behind a paywall, I would suggest to change all links from the webpage / wiki docs to the arxiv version, which is open access and anyone can see it quickly. (or put both).

@vorugantia - Can you please do it?
